src/components/home/Faq.js

```diff
@@ -1,10 +1,105 @@
+'use client';
+import { useState } from 'react';
 import Container from '../shared/Container';
 
+const FAQs = [
+  {
+    question: 'How do I create a restaurant account?',
+    answer:
+      "To create a restaurant account, click on 'Sign Up', choose 'Restaurant Owner', and fill out the required information.",
+  },
+  {
+    question: 'Can I update my restaurant menu?',
+    answer:
+      "Yes, log in to your account and select 'Edit Menu' to update your restaurant menu.",
+  },
+  {
+    question: 'How do I track my food order?',
+    answer:
+      "You can track your order in real-time from the 'Order History' section of your account.",
+  },
+  {
+    question: 'What payment methods are accepted?',
+    answer:
+      'We accept credit/debit cards, PayPal, Google Pay, and cash on delivery.',
+  },
+  {
+    question: 'How can I contact customer support?',
+    answer:
+      'You can reach our customer support team at support@example.com or call +1-123-456-7890.',
+  },
+];
+
 const Faq = () => {
+  // State to track the open/closed state of each FAQ
+  const [faqStates, setFaqStates] = useState(Array(FAQs.length).fill(false));
+
+  // Function which will toggle the open/closed state of a FAQ item
+  const toggleFaq = (index) => {
+    const newFaqStates = [...faqStates];
+    newFaqStates[index] = !newFaqStates[index];
+    setFaqStates(newFaqStates);
+  };
```

Comment:
In the future all faqs come from the backend cause, the website owner does not know how to add a new faq, then that time he can solve this problem using API, one API for faq add in the database and another API for getting faq from the database so here in future we need to data fetching for faqs from the database, that time we use this component as a server component. So create a file name SingleFaq.js and move this code to SingleFaq.js file. In that file we can use 'use client' and useState and other stuff. and remove 'use client' from this file Faq.js file.